modules/front-end/src/app/features/safe/organizations/profile/profile.component.ts

```diff
@@ -52,6 +52,12 @@ export class ProfileComponent {
     });
   }
 
+  ngOnInit(): void {
+    this.resetPasswordForm.get('newPassword').valueChanges.subscribe(() => {
+      this.validateConfirmPassword();
+    });
+  }
+
```

Comment:
🐛 

This component didn't implement `OnInit`. I recommend you let `ProfileComponent` implements `OnInit` and also put the form initialization code into `OnInit` method.

Reply:
Resolved !